pkg/components/wfbackend/wfbackend_component.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package wfbackend
+
+import (
+	"github.com/dapr/kit/logger"
+)
+
+func NewWorkflowBackendComp(logger logger.Logger) WorkflowBackend {
+	return &workflowBackendComponent{
+		logger: logger,
+	}
+}
+
+type workflowBackendComponent struct {
+	logger logger.Logger
+}
+
+func (c *workflowBackendComponent) Init(metadata Metadata) error {
+	c.logger.Info("Initializing Dapr workflow backend component ")
```

Comment:
Missing something?

Reply:
Its basically like [workflow engine components init](https://github.com/dapr/dapr/blob/9e2bcdc3d8a64b02466a9274e9259f166eedc95a/pkg/runtime/wfengine/component.go#L73-L76), which doesn't have any business logic but only to be initialized. In the same way, we are initializing workflow backend component and logging the message.

Reply:
I meant to say that this looks like it's missing something as the error message ends with a space! What is this used for btw?

Reply:
Fixed now.